conf/storm.yaml.example

```diff
@@ -39,10 +39,13 @@
 #     - "server2"
 
 ## Metrics Consumers
+## NOTE: task queue will be unbounded when max.retain.metric.tuples is equal or less than 0.
 # topology.metrics.consumer.register:
 #   - class: "org.apache.storm.metric.LoggingMetricsConsumer"
+#     max.retain.metric.tuples: 100
 #     parallelism.hint: 1
 #   - class: "org.mycompany.MyMetricsConsumer"
+#     max.retain.metric.tuples: 100
```

Comment:
It would be helpful to have an example of how to specify include/exclude filters (whitelist/blacklist).